CSC322-Grinnell/mayflower-directory
# Change default photo

Resolves #165 

# Changes

- Change default profile photo so it doesn't stand out as much against normal profile photos

# Testing

1. If you don't have any dummy profiles yet, run `rails db:dummy_seed`
2. Start the server
3. Make sure the pictures page uses the new silhouette profile photo
